object-storage-core/src/main/java/io/micronaut/objectstorage/UploadRequest.java

```diff
@@ -139,9 +134,6 @@ public Optional<Long> getContentSize() {
             try {
                 return Optional.of(Files.size(path));
             } catch (IOException e) {
-                if (LOG.isErrorEnabled()) {
```

Comment:
I'd keep this one since without it there's no information about why the optional is empty